### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by vlazar**

### CODE REVIEW

**Review by paulcsmith — COMMENTED** [review]
Hi @vlazar!

It looks like the previous change was correct. On line 100 it has `plain_text "Requested user id: \#{some_user_id}"` so the response would be `Request used id: 123-foo`.

I'm going to go ahead and close this, but if you have a suggestion for clarify this, please let me know

**Comment by vlazar** [comment]
@paulcsmith Yes, the response would be `Request used id: 123-foo` but `some_user_id` would return `123-foo`, not `Request used id: 123-foo` as mentioned currently.

**Review by paulcsmith — COMMENTED** [review]
Oh I see! I misunderstood. Nice catch :)

File: src/actions/guides/http_and_routing/routing_and_params.cr [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
     ```
 
     Here, the string from the request path will be returned by the `some_user_id` method. So in this example if 
-    `/users/123-foo` is requested `some_user_id` would return a text response of `Requested user id: 123-foo`.
+    `/users/123-foo` is requested `some_user_id` would return `123-foo`.
```
Comment (jwoertink) [inline src/actions/guides/http_and_routing/routing_and_params.cr]: Can you also add that the action would return a text response of `Requested user id: 123-foo`? I'm thinking something like 

> `some_user_id` would return `123-foo`, and the action would return a text response of `Requested user id: 123-foo`

Comment (paulcsmith) [reply src/actions/guides/http_and_routing/routing_and_params.cr]: Good idea. I think that would help clarify things

Comment (vlazar) [reply src/actions/guides/http_and_routing/routing_and_params.cr]: Updated.

I also wonder if

> would return `123-foo`

Should be

> `some_user_id` would return `"123-foo"`

**Review by jwoertink — APPROVED** [review]
yup, I like it. Thanks!

**Comment by paulcsmith** [comment]
Nice! :D

**Pull request merged**